exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserServer.java

```diff
@@ -343,10 +346,10 @@ public BitToUserHandshake getHandshakeResponse(UserToBitHandshake inbound) throw
           if (inbound.getRpcVersion() != UserRpcConfig.RPC_VERSION) {
             final String errMsg = String.format("Invalid rpc version. Expected %d, actual %d.",
                 UserRpcConfig.RPC_VERSION, inbound.getRpcVersion());
-
             return handleFailure(respBuilder, HandshakeStatus.RPC_VERSION_MISMATCH, errMsg, null);
           }
 
+          Map<String, String> sessionParams = new LinkedHashMap<String, String>();
```

Comment:
May be it's better to keep setUser method as is, I don't really like the idea of creating empty map each time we getHandshakeResponce